ArcGISRuntimeSDKQt_CppSamples/Analysis/StatisticalQueryGroupSort/StatisticalQueryGroupSort.cpp

```diff
@@ -93,43 +93,43 @@ void StatisticalQueryGroupSort::connectSignals()
     m_resultsModel->clear();
     addResultToModel("", QString("Error. %1").arg(error.message()));
   });
+}
 
-  connect(m_featureTable, &ServiceFeatureTable::queryStatisticsCompleted, this, [this](const QUuid&, StatisticsQueryResult* rawResult)
-  {
-    // Delete rawResult when we leave local scope.
-    auto result = std::unique_ptr<StatisticsQueryResult>(rawResult);
+void StatisticalQueryGroupSort::onQueryStatisticsCompleted_(StatisticsQueryResult* rawResult)
+{
+  // Delete rawResult when we leave local scope.
+  auto result = std::unique_ptr<StatisticsQueryResult>(rawResult);
 
-    if (!result)
-      return;
+  if (!result)
+    return;
 
-    // clear previous results
-    m_resultsModel->clear();
+  // clear previous results
+  m_resultsModel->clear();
 
-    // iterate the results and add to a model
-    StatisticRecordIterator iter = result->iterator();
-    while (iter.hasNext())
-    {
-      // get the statistic record
-      StatisticRecord* record = iter.next();
+  // iterate the results and add to a model
+  StatisticRecordIterator iter = result->iterator();
+  while (iter.hasNext())
+  {
+    // get the statistic record
+    StatisticRecord* record = iter.next();
 
-      // get the group string
-      QStringList sectionStrings;
-      const QVariantMap& groupsMap = record->group();
-      for (auto it = groupsMap.cbegin(); it != groupsMap.cend(); ++it)
-      {
-          sectionStrings << QString("\"%1\":\"%2\"").arg(it.key(), it.value().toString());
-      }
-      const QString sectionString = sectionStrings.join(',');
+    // get the group string
+    QStringList sectionStrings;
+    const QVariantMap& groupsMap = record->group();
+    for (auto it = groupsMap.cbegin(); it != groupsMap.cend(); ++it)
+    {
+        sectionStrings << QString("\"%1\":\"%2\"").arg(it.key(), it.value().toString());
+    }
```

Comment:
```suggestion
    for (auto it = groupsMap.cbegin(); it != groupsMap.cend(); ++it)
    {
      sectionStrings << QString("\"%1\":\"%2\"").arg(it.key(), it.value().toString());
    }
```

This is old code, but let's fix it up while we're here.